### PR TITLE
#7632 added nightly file upload integration tests

### PR DESCRIPTION
**Pull request by wal-jan**
Suggestion for nightly to run FileUpload integration tests in Chrome, Firefox with Mojarra, Myfaces, Commons, and Native mode.

### CODE REVIEW

**Comment by tandraschko** [comment]
@christophs78 should we do it like this and create a workflow only fileupload?

File: .github/workflows/nightly-file.upload.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+name: IT FileUpload
+
+on: [push, pull_request]
+  #schedule:
+  #  - cron: '0 8 * * *'
+
+jobs:
+  nightly:
+    if: github.repository == 'primefaces/primefaces' && github.ref == 'refs/heads/master'
```
Comment (christophs78) [inline .github/workflows/nightly-file.upload.yml]: please disable this line temporary to check to workflow-run

**Comment by christophs78** [comment]
> @christophs78 should we do it like this and create a workflow only fileupload?

I need to think about it. :-)
Maybe it´s worth the "price" compared to having an XXL-extensive matrix for checking all combinations as part of our primary IT-matrix.

**Comment by christophs78** [comment]
I´m fine with this approach. IMO we don´t want to add this to our full IT-matrix. So it´s ok, to add another scheduled job to run file upload integration tests with this additional upload-specific-matrix.

**Pull request merged**